src/index.js

```diff
@@ -136,7 +136,6 @@ function generateAndSendSchema(envSecret) {
       liana: configStore.Implementation.getLianaName(),
       liana_version: configStore.Implementation.getLianaVersion(),
       engine: 'nodejs',
-      engine_version: process.versions && process.versions.node,
```

Comment:
This change looks a bit naive to me and won't solve the problem on the long term.
What if the `orm_version`, `database_type`, ... is different from one environment to another?
Those scenarios are as possible as a different engine version.

Partially removing the meta data won't solve the issue and will make Forest lose data.
If the issue you try to solve is real, you'll have to make a choice between `meta` section removal in the schema or `meta` isolation for `apimapHash` computation.